ephios/templates/base.html

```diff
@@ -1,22 +1,36 @@
 {% load i18n %}
 {% load bootstrap4 %}
 {% load static %}
+{% load compress %}
+
 
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>ephios - {% block title %} ephios {% endblock %}</title>
-    <link type="text/css" media="screen" rel="stylesheet" href="{% static "bootstrap/css/bootstrap.min.css" %}">
-    <link type="text/css" media="screen" rel="stylesheet" href="{% static "fontawesome/css/all.css" %}">
-    <link type="text/css" media="screen" rel="stylesheet" href="{% static "ephios/css/main.css" %}">
-    <link type="text/css" media="screen" rel="stylesheet" href="{% static "datatables/datatables.min.css" %}">
-    <link type="text/css" media="screen" rel="stylesheet" href="{% static "select2/css/select2.min.css" %}">
-    {% block css %}{% endblock %}
+    {% compress css %}
+        <link type="text/css" media="screen" rel="stylesheet" href="{% static "bootstrap/css/bootstrap.min.css" %}">
+        <link type="text/css" media="screen" rel="stylesheet" href="{% static "fontawesome/css/all.css" %}">
+        <link type="text/css" media="screen" rel="stylesheet" href="{% static "ephios/css/main.css" %}">
+        <link type="text/css" media="screen" rel="stylesheet" href="{% static "datatables/datatables.min.css" %}">
+        <link type="text/css" media="screen" rel="stylesheet" href="{% static "select2/css/select2.min.css" %}">
+        {% block css %}{% endblock %}
+    {% endcompress %}
+    {% compress js %}
+        <script type="text/javascript" src="{% static "js/jquery.min.js" %}"></script>
+        <script type="text/javascript" src="{% static "bootstrap/js/bootstrap.bundle.min.js" %}"></script>
+        <script type="text/javascript" src="{% static "select2/js/select2.js" %}"></script>
+        <script type="text/javascript" src="{% static "django_select2/django_select2.js" %}"></script>
+        <script type="text/javascript" src="{% static "datatables/datatables.min.js" %}"></script>
+        <script type="text/javascript" src="{% static "js/jquery.formset.min.js" %}"></script>
```

Comment:
As far as I understand, we don't need to use the min versions here, as they get compressed anyway...